packages/docusaurus-config/lib/routing.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file Docusaurus routing configuration.
+ */
+
+/**
+ * @typedef {'production'|'development'} NodeEnvironment
+ */
+
+const NodeEnvironment = /** @type {NodeEnvironment} */ (process.env.NODE_ENV || "development");
+
+/**
+ * @satisfies {Record<NodeEnvironment, Record<string, string>>}
+ */
+export const DocusaurusURLByEnvironment = /** @type {const} */ ({
+    development: {
+        Docs: "http://localhost:3000",
+        Integrations: "http://localhost:3001",
+        WWW: "http://localhost:3002",
+    },
+    production: {
+        Docs: "https://docs.goauthentik.io",
+        Integrations: "https://docs.goauthentik.io/integrations",
+        WWW: "https://goauthentik.io",
+    },
+});
+
+export const DocusaurusURL = DocusaurusURLByEnvironment[NodeEnvironment];
+
+/**
+ * @satisfies {Record<string, string>}
+ */
+export const SocialURL = /** @type {const} */ ({
+    Discord: "https://discord.goauthentik.io",
```

Comment:
```suggestion
    Discord: "https://goauthentik.io/discord",
```

Shouldn't this be the right url